virttest/libvirt_vm.py

```diff
@@ -474,6 +475,17 @@ def add_mem(help_text, mem, maxmem=None, hugepage=False):
                                         "virt-install")
                     else:
                         cmd += ",hugepages=yes"
+                if hotplugmaxmem:
+                    if not has_sub_option('memory', 'hotplugmemorymax'):
+                        logging.warning("hotplugmemorymax option not supported"
+                                        "by virt-install")
+                    else:
+                        cmd += ",hotplugmemorymax=%s" % hotplugmaxmem
+                    if not has_sub_option('memory', 'hotplugmemoryslots'):
+                        logging.warning("hotplugmemoryslots option not "
+                                        "supported by virt-install")
+                    else:
+                        cmd += ",hotplugmemoryslots=%d" % int(hotplugmemslots)
```

Comment:
type cast is not necessary as you have already taken care of it during assignment.

Reply:
sure, will update it.